ansible_image_validation/validation-playbooks/ade_packages_validation.yaml

```diff
@@ -0,0 +1,45 @@
+###
+# This Playbook checks if the packages required by Azure Disk Encryption is installed 
+###
+
+---
+
+- name: Check if the packages required by Azure Disk Encryption is installed
+  command: rpm -q "{{ item }}"
+  loop:
+    - lsscsi
+    - psmisc
+    - lvm2
+    - uuid
+    - at
+    - patch
+    - cryptsetup
+    - cryptsetup-reencrypt
+    - procps-ng
+    - util-linux
+  register: package_check
+  ignore_errors: yes
+
+- name: "Write to error msg if ADE packages are not present"
+  lineinfile:
+    path: "{{err_folder}}/err_msgs.log"
+    line: "packages required for ADE are not present"
+    create: yes
+    state: present
+  when: package_check is not succeeded
```

Comment:
This can be redirected at the earlier itself if followed my suggestion

Reply:
@sribs  can you try running your logic with some packages in the loop and share the snippet here? It would be a great help, Thanks